libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -78,7 +78,16 @@ class ofEasyCam : public ofCamera {
     void setAutoDistance(bool bAutoDistance);
 
     void setEvents(ofCoreEvents & events);
-
+	
+	/// \brief Set the input sensitivity of the rotation.
+	/// X and Y axes - when the value is 1.0, moving the mouse from one side to the other of the arcball (min(viewport.width, viewport.height)) will rotate 180 degrees. When the value is 0.5, the rotation will be 90 degrees.
```

Comment:
Thanks for documenting!

Please add hard line breaks to documentation @ 80 chars for readability.

https://github.com/openframeworks/openFrameworks/wiki/openFrameworks-doxygen-documentation-style-guidelines#document-the-class